GOVERNANCE.md

```diff
@@ -1,66 +1,119 @@
-* In general disputes about code should be resolved by discussion, not votes. If there is substantial disagreement, aim for consensus. If that is not possible, committers can bring the dispute to the TC for a vote, with the outcome determined by a majority.
-* Any committer can decide to close a PR or issue if they determine the change doesn't suit the project.
-* WebdriverIO has a large ecosystem of repositories. Some are 'core' in the sense of being central subpackages with a lot of traffic like `webdriverio/webdriverio`. Others are 'peripheral' in the sense of receiving few changes, being maintained by 1 person or no one, etc..., like `webdriverio/cucumber-boilerplate` or `webdriverio/jasmine-boilerplate`. For 'core' repos, committers should not merge their own code straightaway. Like all contributors they should open a PR and get a +1 from another committer. For 'peripheral' repos for which they are the sole maintainer or for which they have a good understanding, it is obtuse to insist on another contributor's +1 and this rule should be relaxed in that case. In sum: use good judgment and don't ram code through without a review when you can help it.
-
-The current committers are (in addition to the TC members above):
-
-* [@amitaibu](https://github.com/amitaibu)
-* [@andriilazebnyi](https://github.com/andriilazebnyi)
-* [@BorisOsipov](https://github.com/BorisOsipov)
-* [@caoxu2000](https://github.com/caoxu2000)
-* [@erwinheitzman](https://github.com/erwinheitzman)
-* [@fijijavis](https://github.com/fijijavis)
-* [@FloValence](https://github.com/FloValence)
-* [@georgecrawford](https://github.com/georgecrawford)
-* [@goofballLogic](https://github.com/goofballLogic)
-* [@jankcat](https://github.com/jankcat)
-* [@klamping](https://github.com/klamping)
-* [@lgandecki](https://github.com/lgandecki)
-* [@tborys](https://github.com/tborys)
-* [@TheSavior](https://github.com/TheSavior)
-* [@ThomasFett](https://github.com/ThomasFett)
-* [@TuHuynhVan](https://github.com/TuHuynhVan)
-* [@WillBrock](https://github.com/WillBrock)
-* [@wswebcreation](https://github.com/wswebcreation)
-* [@wvankuipers](https://github.com/wvankuipers)
-* [@zinserjan](https://github.com/zinserjan)
+* In general any committer can review and merge a PR. In general committers should only merge code they are qualified to review, which might entail pinging another committer who has greater ownership over a specific code area.
```

Comment:
This contradicts the lines above about TSC members merging, and the requirement for 3 approvals, and the line below about work reviewed by TSC members.

Reply:
I think this section is about bug fixes, doc changes or other smaller contributions. A requirement for 3 approvals only applies for substantial changes, e.g. features like spec level retries